.eslintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
 				'plugin:@typescript-eslint/recommended',
 			],
 			rules: {
+				camelcase: 'off',
```

Comment:
This rule won't work well with any snake case data we get from the server and add types for. Example in this case was the profile items data. in future we could camelCase convert everything we get from the server, for now switching this off on TS makes sense imo.

Reply:
@louwie17 
> I wasn't a huge fan of graphql

Thats tragic, guess you're not a 10xer unicorn then smh 😛 

> I guess something we would have to do client side

Yeah i imagine a function that transforms the props to camel case / snake case when the data comes in / goes out. Its probably not a huge deal to solve at this point though.